_news-items/2024-08-16-collins.html

```diff
@@ -0,0 +1,35 @@
+---
+# Copyright 2024 seL4 Project a Series of LF Projects, LLC.
+# SPDX-License-Identifier: CC-BY-SA-4.0
+title: Welcome Collins, Bronze sponsor of the seL4 Summit 2024
+anchor: sponsor24-collins
```

Comment:
can leave this one out for new items

Reply:
But we need the title and anchor, don't we?

Reply:
Removing these lines also results in a licence error. Pls see below.

Reply:
I meant just the `anchor` line, the rest we do need.

Reply:
(Should have realised that GitHub will show more lines of context -- sorry for the confusion)

Reply:
Oh righto. I'll fix it now. Thanks for letting me know!